src/roslyn/src/NuGet/Microsoft.Net.BuildServerUtils/Microsoft.Net.BuildServerUtils.csproj

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetRoslynSourceBuild)</TargetFrameworks>
+
+    <!-- NuGet -->
+    <IsPackable>true</IsPackable>
+    <IsSourcePackage>true</IsSourcePackage>
```

Comment:
This is a new source package which can be used by our build servers (currently used by roslyn, but should be also used by msbuild and razor). Lives in Roslyn because that flows to all the other repos (but I'm open to other ideas).